smurfs/src/actions/index.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios'
+
+export const START_FETCHING = 'START_FETCHING';
+export const FETCH_SUCCESS = 'FETCH_SUCCESS';
+export const FETCH_FAILURE = 'FETCH_FAILURE';
+
+export const START_POST = 'START_POST';
+export const POST_SUCCESS = 'POST_SUCCESS';
+export const POST_FAILURE = 'POST_FAILURE';
```

Comment:
I've seen action creators set up two different ways - One, where all of the action types are at the top of the file with the functions down below (like your code here), and another where the action types are right above the function itself, something like so:
```
export const START_POST = 'START_POST';
export const POST_SUCCESS = 'POST_SUCCESS';
export const POST_FAILURE = 'POST_FAILURE';
export const postSmurf = (newSmurf) => dispatch => {
    dispatch({ type: START_POST });
    axios
        .post(`http://localhost:3333/smurfs`, newSmurf)
        .then(res => dispatch({ type: POST_SUCCESS, payload: res.data }))
        .catch(err => dispatch({ type: POST_FAILURE, payload: err.response}));
};
```
How you organize the file is up to you - as long as everything is getting exported then you're good to go 👍